src/foam/nanos/auth/Phone.js

```diff
@@ -17,7 +17,14 @@ foam.CLASS({
     {
       class: 'PhoneNumber',
       name: 'number',
-      required: true
+      required: true,
+      validateObj: function (number) {
+        var numberRegex = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
```

Comment:
This doesn't look like it would capture if someone were to add their country code to the beginning. i.e. +14169000000